CHANGELOG.md

```diff
@@ -15,15 +15,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Handle body rotation on physics integration (#1242, **&fallenatlas**).
 - Binary Serializer and Deserializer (#1306, **@RiscadoA**).
 - Type Client and Type Server (#1302, **@RiscadoA**).
-- Deadzone for input axis (#844, **@kuukitenshi**)
-- Generic Camera component to hold projection matrix (#1331, **@mkuritsu**)
+- Deadzone for input axis (#844, **@kuukitenshi**).
+- Generic Camera component to hold projection matrix (#1331, **@mkuritsu**).
 - Initial application debugging through Tesseratos (#1303, **@RiscadoA**).
 - Print stacktrace with *cpptrace* on calls to CUBOS_FAIL (#1172, **@RiscadoA**).
 
 ### Fixed
 
 - Spot light angle mismatch between light and shadows (#1310, **@tomas7770**).
 - Spot shadows cause light range cutoff (#1312, **@tomas7770**).
+- Moved Glad and stb-image libs to another repositories, cubos-glad and cubos-stb, respectively (#1323, **@kuukitenshi**).
```

Comment:
I would not call this a fix - you can add it to a new `### Changed` section, between the Added and Fixed sections :relieved: (we're following this changelog format btw: https://keepachangelog.com/en/1.1.0/)